### DISCUSSION TITLE
Preload

### DISCUSSION THREAD

**Discussion by esginmurat** [bootscore/bootscore#522] [General]
Hello. Hi @crftwrk First of all, thank you again for this great theme.
I don't understand the "online" tag in the fontawesome preloading section in functions.php. By mistake? or should it be "onload"? Thank you from now.

```
// Preload Font Awesome
add_filter('style_loader_tag', 'bootscore_fa_preload');

function bootscore_fa_preload($tag) {

  $tag = preg_replace("/id='fontawesome-css'/", "id='fontawesome-css' online=\"if(media!='all')media='all'\"", $tag);

  return $tag;
}

// Preload Font Awesome END
```

Also, is it about the narratives in the article that is aimed here?
https://web.dev/preload-critical-assets/
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/preload

Regards..

**Comment by crftwrk** [comment]
Hm, to be honest, I have no idea. This script is very very old but seems to work fine? Want to change this quickly and create a pull request?

**Reply by esginmurat** [reply]
Hello. Thank you for the answer.
I don't know how to do "pull request". However, I did my research. As far as I understand, the "online" element has no effect on the code here.

**Reply by crftwrk** [reply]
Hi Murat,

a pull request is a simple tool which allows you to doing changes without unhandy reporting issues to maintainers. This is how community-driven open source projects works. The preload snippet is a good example to create your first "PR", you will see, it's fast, easy and makes a lot of fun.

- Go to https://github.com/bootscore/bootscore and press the "Fork" button on the top right.
- Now you have a fork (a copy) of the theme in your own profile where you can edit files as you need.
- Change preload snippet in your fork.
- Once done, GitHub will ask you if you want to contribute and create a pull request. Choose yes, done.
- Now your changes are requested here in the main repo and we can simply merge them.

 All this is done in less than 2 minutes. If you need help, you can ask me anytime.
